Misc/NEWS.d/next/Library/2020-02-22-12-02-11.bpo-39716.z2WhDQ.rst

```diff
@@ -0,0 +1,3 @@
+Raise an ArgumentError when the same subparser name is added twice to an
```

Comment:
How about using `added twice or more` to replace `added twice`?

Reply:
You won't be able to add it more than twice if it errors out on the second addition...

Reply:
make sense.

@rhettinger Hi, raymond. Pls review this PR if you have free time.